plugin/helpers_bots.go

```diff
@@ -72,3 +91,37 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 
 	return createdBot.UserId, nil
 }
+
+// EnsureBotWithOptions calls EnsureBot with the specified Options,
+// currently supporting paths to the bot profile image and/or icon image
+func (p *HelpersImpl) EnsureBotWithOptions(bot *model.Bot, setters ...Option) (retBotId string, retErr error) {
+	// Default options
+	args := &ensureBotOptions{
+		ProfileImagePath: "",
+		IconImagePath:    "",
+	}
+	for _, setter := range setters {
+		setter(args)
+	}
+	botId, err := p.EnsureBot(bot)
+	if err != nil {
+		return botId, errors.Wrap(err, "Failed to get or create bot")
+	}
+
+	if !(args.ProfileImagePath == "") {
+		bytes, err := p.API.ReadFile(args.ProfileImagePath)
```

Comment:
`ReadFile` tried to open the file from the `data` directory of the server. Hence we can't use it here. We have to do something like:
```go
	bundlePath, err := p.API.GetBundlePath()
	if err != nil {
		return errors.Wrap(err, "failed to get bundle path")
	}

	profileImage, err := ioutil.ReadFile(filepath.Join(bundlePath, args.ProfileImagePath))
	if err != nil {
		return errors.Wrap(err, "failed to read profile image")
	}
	if appErr := p.API.SetProfileImage(botId, profileImage); appErr != nil {
		return errors.Wrap(appErr, "failed to set profile image")
	}
```